packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx

```diff
@@ -56,12 +56,16 @@ interface Props {
 const Bech32Banner = (props: Props) => {
     const theme = useTheme();
     return (
-        <StyledCard>
+        <StyledCard data-test="@accounts/new/modal">
```

Comment:
```suggestion
        <StyledCard data-test="@accounts/empty-account/default-native-account">
```

Reply:
@tsusanka this is confusing, that it is called Bech32Banner but there is something about default accounts.

Reply:
534889de5464e01a3eb5970394719506dc079b1e done

Reply:
@tomasklim you are right but that's probably out of scope for this PR..